typings/embedded_io/audioout.d.ts

```diff
@@ -0,0 +1,49 @@
+/*
+* Copyright (c) 2025 Satoshi Tanaka
+*
+*   This file is part of the Moddable SDK Tools.
+*
+*   The Moddable SDK Tools is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   The Moddable SDK Tools is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with the Moddable SDK Tools.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+declare module "embedded:io/audio/out" {
+  import { Buffer } from "embedded:io/_common";
+
+  type AudioType = "LPCM";
+
+  interface AudioOutOptions {
+    bitsPerSample?: 8 | 16;
+    channels?: 1 | 2;
+    sampleRate?: number;
+    audioType?: AudioType;
+    format?: "buffer";
+  }
+
+  class AudioOut {
+    constructor(options?: AudioOutOptions);
```

Comment:
This should include `onWritable(byteLength, sampleCount)` – see [Callbacks in Audio Output specification](https://ecmatc53.github.io/spec/web/spec.html#-10-io-classes-audio-output-synchronous-io).